src/DotNetLightning.Core/Channel/CommitmentsModule.fs

```diff
@@ -409,10 +424,13 @@ module internal Commitments =
                                  |> Map.toSeq |> Seq.map (fun (k, _) -> k) |> Set.ofSeq
                         Set.difference t1 t2
                     let originChannels1 = cm.OriginChannels |> Map.filter(fun k _ -> Set.contains k completedOutgoingHTLCs)
-                    { cm with LocalCommit = localCommit1
-                              LocalChanges = ourChanges1
-                              RemoteChanges = theirChanges1
-                              OriginChannels = originChannels1 }
+                    {
+                        cm with
+                        LocalCommit = localCommit1
+                        LocalChanges = ourChanges1
+                        RemoteChanges = theirChanges1
+                        OriginChannels = originChannels1
+                    }
```

Comment:
you're not really changing the lines above, so it's not worth it to change the style